tests/ruby-sass/media-width.scss

```diff
@@ -0,0 +1,37 @@
+@import "../setup";
+
+.media-width-from {
+  @include wtr-media-width(from, 250px) {
+    color: red
+  }
+}
+
+.media-width-until {
+  @include wtr-media-width(until, 250px) {
+    color: blue
+  }
+}
+
+.media-width-equal {
+  @include wtr-media-width(equal, 250px) {
+    color: lime
```

Comment:
Color `lime` should be written in hexadecimal form as `#00ff00`<br>Declaration should be terminated by a semicolon